bootstrap/gamma/deploy_native_test.go

```diff
@@ -17,34 +17,41 @@ import (
 	"time"
 )
 
-func TestNonLeaderDeploysNativeContract(t *testing.T) {
-	if testing.Short() {
-		t.Skip("Skipping compilation of contracts in short mode")
-	}
+func testDeployNativeContractWithConfig(jsonConfig string) func(t *testing.T) {
+	return func(t *testing.T) {
+		test.WithContext(func(ctx context.Context) {
+			network := NewDevelopmentNetwork(ctx, log.DefaultTestingLogger(t), jsonConfig)
+			contract := callcontract.NewContractClient(network)
 
-	test.WithContext(func(ctx context.Context) {
-		network := NewDevelopmentNetwork(ctx, log.DefaultTestingLogger(t), "")
-		contract := callcontract.NewContractClient(network)
+			counterStart := contracts.MOCK_COUNTER_CONTRACT_START_FROM
 
-		counterStart := contracts.MOCK_COUNTER_CONTRACT_START_FROM
+			t.Log("deploying contract")
 
-		t.Log("deploying contract")
+			contract.DeployNativeCounterContract(ctx, 1, 0) // leader is nodeIndex 0, validator is nodeIndex 1
```

Comment:
Nitpicking - the comment about "leader is 0 ..." is correct only for benchmark consensus and for the initial view of lean helix.